boofuzz/sessions.py

```diff
@@ -1043,8 +1045,8 @@ def _restart_target(self, target):
                 self._fuzz_data_logger.log_info("Restarting target process using {}".format(monitor.__class__.__name__))
                 if monitor.restart_target(target=target, fuzz_data_logger=self._fuzz_data_logger, session=self):
                     # TODO: doesn't this belong in the process monitor?
-                    self._fuzz_data_logger.log_info("Giving the process 3 seconds to settle in")
-                    time.sleep(3)
+                    self._fuzz_data_logger.log_info("Giving the process %d seconds to settle in" % self.restart_wait)
+                    time.sleep(self.restart_wait)
```

Comment:
I'm not 100% happy with adding yet another Session argument. How about reusing `restart_sleep_time`? It is currently used to simply sleep for the given time if no other reset handler is available.

Also, as the TODO suggests, the logic to get the target task up and running belongs in the process monitor not in Session. Nothing that has to be done here, but another argument against adding the parameter to Session.

Reply:
I like the idea of adding the arg to the procmon best, but I'm OK with an arg in Session since it's better than nothing. I agree with @SR4ven's idea of reusing `restart_sleep_time`.